packages/read-model/src/Product/Listed/ListedProductViewFactory.php

```diff
@@ -144,12 +145,20 @@ public function createFromProduct(Product $product, ?ImageView $imageView, Produ
      */
     public function createFromArray(array $productArray, ?ImageView $imageView, ProductActionView $productActionView, PricingGroup $pricingGroup): ListedProductView
     {
+        $productPrice = $this->priceFactory->createProductPriceFromArrayByPricingGroup(
```

Comment:
can you please add deprecation notice into method `createProductPriceFromArrayByPricingGroup` when trying to return null as we discussed?

Reply:
Sure. It has been done in 822e9bbdbf1f19011910b9675d4b963e2d785040